system/library/helperbm.php

```diff
@@ -148,6 +152,14 @@ public function getInvoiceFeeTax()
         return  $this->config->get('payment_billmate_checkout_inv_fee_tax');
     }
 
+    /**
+     * @return string
+     */
+    public function isCompanyView()
+    {
+        return  (bool)$this->config->get('payment_billmate_checkout_is_company_view');
```

Comment:
Should be: 

`return (bool)$this->config->get('payment_billmate_checkout_is_company_view');`

Reply:
Are you sure that it is the best way to write this row? Which doc describes that we need to write it in this way?

Reply:
Oh, I got, it is about two white spaces.